tests/e2e/multicontrolplane/multi_control_plane_suite_test.go

```diff
@@ -30,9 +30,10 @@ import (
 )
 
 var (
-	cl                     client.Client
-	err                    error
-	version                = istioversion.New
+	cl  client.Client
+	err error
+	// version will be the first version in the list, this version is the newest Istio version in the versions.yaml file
+	version                = istioversion.List[0].Name
```

Comment:
Why this change? Isn't this the same as `istioversion.New` that was here before?